### PR TITLE
Deployment

### PR DESCRIPTION
**Pull request by elbotho**
- https://github.com/serlo/frontend/pull/3410
- https://github.com/serlo/frontend/pull/3353
- https://github.com/serlo/frontend/pull/3330
- https://github.com/serlo/frontend/issues/3418
- #3422

### CODE REVIEW

**Comment by Entkenntnis** [comment]
@elbotho can we deploy this today?

File: apps/web/src/components/entity-base.tsx [hunk 1]
```diff
@@ -106,37 +106,45 @@ export function EntityBase({ children, page, entityId }: EntityBaseProps) {
...
+                  </button>
                 </div>
               </div>
             </a>
-            <div className="hidden sm:block">{renderUnicorn()}</div>
+
+            {/* eslint-disable-next-line @next/next/no-img-element */}
+            <img
+              className="my-8 mr-8 hidden h-20 mobileExt:h-40 sm:mb-10 sm:ml-4 sm:mt-6 sm:block sm:h-48"
+              src="https://denkspace.de/images/tina.png"
```
Comment (elbotho) [inline apps/web/src/components/entity-base.tsx]: I don't love getting assets from another server here. Also not sure if @yanncloud want's the traffic ;)

File: apps/web/src/components/entity-base.tsx [hunk 2]
```diff
@@ -230,21 +238,23 @@ export function EntityBase({ children, page, entityId }: EntityBaseProps) {
...
-    if (!asPath.startsWith('/mathe/')) {
-      return
-    }
+    const isMath =
+      asPath.startsWith('/mathe/') ||
+      (page.breadcrumbsData &&
+        page.breadcrumbsData.some((val) => val.label.includes('Mathematik')))
 
-    if (asPath.includes('einhorn-der-mathematik')) {
+    console.log(isMath, page.breadcrumbsData)
```
Comment (elbotho) [inline apps/web/src/components/entity-base.tsx]: leftover?

**Comment by Entkenntnis** [comment]
Uuid to check after deploy: 1511

File: apps/web/src/components/entity-base.tsx (hunk 2 above)
Comment (Entkenntnis) [reply apps/web/src/components/entity-base.tsx]: removed

File: apps/web/src/components/entity-base.tsx (hunk 1 above)
Comment (Entkenntnis) [reply apps/web/src/components/entity-base.tsx]: uploaded to serlo

**Pull request merged**